manifests/s/StudioRack/StudioRack/0.1.0/StudioRack.StudioRack.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: StudioRack.StudioRack
+PackageVersion: 0.1.0
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```

Reply:
How does this happen everytime? I just use wingetcreate update to update the installer link..